apps/admin-x-activitypub/src/components/modals/NewPostModal.tsx

```diff
@@ -17,23 +17,28 @@ const NewPostModal = NiceModal.create(() => {
 
     const handlePost = async () => {
         const trimmedContent = content.trim();
+
         if (!trimmedContent) {
             return;
         }
 
-        await noteMutation.mutate({content: trimmedContent}, {
+        noteMutation.mutate({content: trimmedContent}, {
```

Comment:
No need to await here, success / error criteria is handled by mutation callbacks